website/docs/r/vpn_site.html.markdown

```diff
@@ -96,6 +98,22 @@ A `link` block supports the following:
 
 * `speed_in_mbps` - (Optional) The speed of the VPN device at the branch location in unit of mbps.
 
+---
+
+A `o365_policy` block supports the following:
+
+* `traffic_category` - (Optional) A `traffic_category` block as defined above.
+
+---
+
+A `traffic_category` block supports the following:
+
+* `allow_controlled` - (Optional) Is allow category controlled? Defaults to `false`.
+
+* `default_controlled` - (Optional) Is default category controlled? Defaults to `false`.
+
+* `optimize_controlled` - (Optional) Is optimize category controlled? Defaults to `false`.
```

Comment:
i'm still not sure what these values are doing? could we eitehr fixuyp the property names or the descriptions?

Reply:
[Here](https://docs.microsoft.com/en-us/microsoft-365/enterprise/urls-and-ip-address-ranges?view=o365-worldwide) are the ip ranges for office that fall within specific Allow/Optimize/Default categories of traffic. What this tells is essentially what kind of traffic from the branch should go directly to Office and what can flow via virtual wan.

[Here](https://docs.microsoft.com/en-us/microsoft-365/enterprise/microsoft-365-network-connectivity-principles?view=o365-worldwide#new-office-365-endpoint-categories) is the defintion of `Allow/Default/Optimize` endpoint.